client/src/api/modules.js

```diff
@@ -3,3 +3,35 @@ const API_URL = 'http://localhost:4000'
 export const getModules = () => {
   return fetch(`${API_URL}/module`).then(response => response.json());
 };
+
+export const createModule = (title) => {
+  return fetch(`${API_URL}/module`, {
+    method: 'POST',
+    headers: new Headers({
+        "Content-Type": "application/json"
+      }),
+    body: JSON.stringify({
+      title: title,
+    })
+  }).then(response => response.json());
+};
+
+export const deleteModule = (id) => {
+  return fetch(`${API_URL}/module/${id}`, {
+    method: 'DELETE'
+  }).then(response => response.json());
+};
+
+export const updateModule = (id, title) => {
+  return fetch(`${API_URL}/module/${id}`, {
+    method: 'PATCH',
+    headers: new Headers({
+        "Content-Type": "application/json"
+      }),
+      title: title,
```

Comment:
```suggestion
      body: JSON.stringify({ title: title }),
```
You are passing `title`, but you use `req.body` in your server-side update function :)

Reply:
Good work on module addition!

Reply:
oh yes I saw it, I just pasted from a tutorial and then forgot it, you sent this code last saturday, I should get it by doing this Module.findOneAndUpdate({ _id: req.params.moduleId }, { title }, { new: true })